data.sql

```diff
@@ -1,5 +1,6 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals VALUES (1, 'Agumon', TO_DATE('03/02/2020', 'DD/MM/YYYY'), 0, true, 10.23);
+INSERT INTO animals VALUES (2, 'Gabumon', TO_DATE('15/11/2018', 'DD/MM/YYYY'), 2, true, 8);
+INSERT INTO animals VALUES (3, 'Pikachu' , TO_DATE('07/01/2021', 'DD/MM/YYYY'), 1, false, 15.04);
+INSERT INTO animals VALUES (4, 'Devimon' , TO_DATE('12/05/2017', 'DD/MM/YYYY'), 5, true, 15.04);
```

Comment:
- Good job done so far, According to the project requirements the weight for `Devimon` should be `11.0` not `15.04`.Please fix this 🙏